src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Area {
```

Comment:
```suggestion
public class Circle extends Figure {
```
let's implement Area interface in the Figure class

Reply:
done